doc/cmake/INSTALL.md

```diff
@@ -13,20 +13,23 @@ brew install wrf-cmake -v
 For more flexibility, e.g. when changing the registry, see the manual build instructions below.
 
 ## Install dependencies
-The following libraries are required on your system to install WRF-CMake from source: [Git](https://git-scm.com/), [JasPer](https://www.ece.uvic.ca/~frodo/jasper/), [libpng](http://www.libpng.org/pub/png/libpng.html), [libjpeg](http://libjpeg.sourceforge.net/), [zlib](https://zlib.net/), [HDF5](https://support.hdfgroup.org/HDF5/), [NetCDF-C](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), [NetCDF-Fortran](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), and MPI (required if building in `dmpar` or `dm_sm` mode). The above libraries are most likely available from your system's package manager (e.g. APT, yum, Homebrew, etc.). If you do not have the latest version of these libraries installed on your system, please see [this page](LIBS.md).
+The following libraries are required on your system to install WRF-CMake from source: [CMake](https://cmake.org/), [Ninja](https://ninja-build.org/), [Git](https://git-scm.com/), [JasPer](https://www.ece.uvic.ca/~frodo/jasper/), [libpng](http://www.libpng.org/pub/png/libpng.html), [libjpeg](http://libjpeg.sourceforge.net/), [zlib](https://zlib.net/), [HDF5](https://support.hdfgroup.org/HDF5/), [NetCDF-C](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), [NetCDF-Fortran](https://www.unidata.ucar.edu/downloads/netcdf/index.jsp), and MPI (required if building in `dmpar` or `dm_sm` mode). The above libraries are most likely available from your system's package manager (e.g. APT, yum, Homebrew, etc.). If you do not have the latest version of these libraries installed on your system, please see [this page](LIBS.md).
 
 ## Build and Install WRF-CMake
 
 ### Transition from original build system
 
-| Original      | CMake          |
-| ------------- | -------------- |
-| `./configure` | `cmake ...`    |
-| `./compile`   | `make install` |
+| Original      | CMake               |
+| ------------- | ------------------- |
+| `./configure` | `cmake ..`          |
+| `./compile`   | `cmake --build .`   |
```

Comment:
Any reason you're separating the two?

Reply:
Do you mean why did I add a new line for install? If so just as compile for Make does not 'fully install' as such so in case you don't want to 'install' you have the option to just build!?